.travis.yml

```diff
@@ -5,6 +5,6 @@ install:
   - composer install
   - bash tests/install-tests.sh wordpress_test root '' 127.0.0.1 latest
 script:
-  - phpunit
+  - phpunit --exclude-group 32656
```

Comment:
Excludes the preflight filters in core because Cavalcades preflight filters interfere cause false failures.